server/db/User.js

```diff
@@ -29,22 +27,26 @@ module.exports.deleteUserByUserId = (userId) => {
 }
 
 module.exports.deleteUserByUsername = (username) => {
-  return db.query(`DELETE FROM users where username = '${username}'`);
+  return db.query(`DELETE FROM users where username='${username}'`);
 }
 
 module.exports.createUser = (username, password, profile) => {
   return db.query(`INSERT INTO users (username, password, profile) VALUES ('${username}', '${password}', '${profile}')`);
 }
 
-module.exports.editUserProfile = (userId, profile) => {
-  return db.query(`UPDATE users SET profile = '${profile}' WHERE id = ${userId};`);
+module.exports.editUserProfile = (username, profile) => {
+  return db.query(`UPDATE users SET profile = '${profile}' WHERE username='${username}';`);
```

Comment:
Only because if we add -> delete -> re-add the same user, the userId will change. Since username is unique, I though eitUserProfile by username makes more sense. We definitely can have two editUserProfile functions. I'm on it. Wait for my next commit.